File: CandidateApp.Domain/Models/Skill.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CandidateApp.Domain.Models
{
    public class Skill
    {
        public int ID { get; set; }
        public string Name { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime UpdatedDate { get; set; }
    }

}

File: CandidateApp.Client/src/app/models/skill.ts
export class Skill {
  id: number;
  name: string;
  createdDate: string;
  updatedDate: string;

  constructor(
    id: number,
    name: string,
    createdDate: string,
    updatedDate: string,
  ) {
    this.id = id;
    this.name = name;
    this.createdDate = createdDate;
    this.updatedDate = updatedDate;
  }
}
